.circleci/config.yml

```diff
@@ -67,6 +67,28 @@ jobs:
             elif ! (curl https://api.github.com/repos/pandas-dev/pandas/issues/$CIRCLE_PR_NUMBER | jq '.labels' | grep -q 'Build'); then
                 circleci-agent step halt
             fi
+      - run:
+          # The process here is more complicated than on Github Actions
+          # since we need to copy the tests wheel into the unzipped sdist
+          # so that it gets copied (along with the contents of the unzipped sdist)
+          # into the Linux docker image used for building the wheel
+          # (unlike Github Actions there is no host access using cibuildwheel with CircleCI)
+          name: Build the sdist
+          command: |
+            pip3 install build setuptools-scm wheel

```

Comment:
Is setuptools-scm a misnomer or does it have some kind of dependency with setuptools?

Reply:
We use it to version the pandas-tests package?

Setting up versioneer is probably overkill for pandas-tests.

Reply:
Yeah, pandas-tests is built using setuptools.

I figured it was much easier to use setuptools given this is a pure Python package.

(Note that if you're just doing regular development you'll never need setuptools, unless you want to build pandas_tests by hand for some reason)